initialization errored: Cannot find module 'lodash'

Can't deploy function, I'm getting this error:


`Serverless plugin "serverless-openwhisk" initialization errored: Cannot find module 'lodash'`

<img width="749" alt="screen shot 2017-11-15 at 2 39 10 pm" src="https://user-images.githubusercontent.com/9260214/32856447-df5a4254-ca12-11e7-838b-a8dfcf0ca073.png">

--- a/package.json
+++ b/package.json
@@ -35,6 +35,7 @@
     "fs-extra": "^1.0.0",
     "get-stdin": "^5.0.1",
     "jszip": "^3.1.3",
+    "lodash": "^4.17.4",
     "moment": "^2.16.0",
     "openwhisk": "^3.4.0"
   },

